setup/gcp/installer/util/constants.py

```diff
@@ -288,6 +288,11 @@ class DeploymentStatus(Enum):
     'https://cloud.google.com/resource-manager/docs/'
     'creating-managing-organization')
 
+MESSAGE_RUN_FREQUENCY = (
+    'Forseti will run once every 12 hours, you can configure the run '
+    'frequency in the server deployment template field "run-frequency"'
+    ' and update the deployment using the deployment manager.')
```

Comment:
nit: just to be consistent, can you please move the whitespace to the line above?

Reply:
Updated.